cloudinit/config/cc_growpart.py

```diff
@@ -433,84 +446,85 @@ def resize_devices(resizer, devices, distro: Distro):
             )
             continue
 
-        try:
-            statret = os.stat(blockdev)
-        except OSError as e:
-            info.append(
-                (
-                    devent,
-                    RESIZE.SKIPPED,
-                    "stat of '%s' failed: %s" % (blockdev, e),
+        if fs != "zfs":
```

Comment:
I do *not* like this.
There has to be a better way, but I'm not yet sure what it is.

Reply:
added a helper function to remove this indent.